nilearn/plotting/surface/tests/test_matplotlib_backend.py

```diff
@@ -14,6 +14,11 @@
     load_surf_mesh,
 )
 
+pytest.importorskip(
+    "matplotlib",
+    reason="Matplotlib is not installed. It is required to run the tests!",
+)
+
```

Comment:
This module will contain only the tests that should be run if `matplotlib` is installed otherwise be skipped.